src/molecules/attendances.module.css

```
@@ -0,0 +1,8 @@
+.attendanceStatus {
+    max-width: 20rem;
+    margin: 0.4rem auto;
+  }
+  
+  .statusItem {
+    padding: 0.2rem;
```

```suggestion
    margin: 6px auto;
  }
  
  .statusItem {
    padding: 3px;
```
これだとフォントサイズを大きくするほど余白がどんどん太くなります. 余白の寸法は固定であってほしいので, 余白の単位には px を使ってほしいです.